client/sites-dashboard/components/link-in-bio-banner/link-in-bio-double-tile-banner.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from '@emotion/styled';
+import image from 'calypso/assets/images/onboarding/link-in-bio-banner-large.png';
+import * as Banner from './link-in-bio-banner-parts';
+
+const Wrapper = styled( 'div' )( {
+	// this is the height of the "footer" div in `SitesGridTile`.
+	// The padding here keeps the bottom of the banner and
+	// the bottom of the site preview/thumbnail aligned
+	paddingBottom: 70.79,
+	gridColumn: 'span 2',
+} );
+
+const Root = styled( Banner.Root )( {
+	alignItems: 'center',
+	padding: '0 64px',
+	paddingInlineEnd: 0,
+	height: '100%',
+	'.dismiss-button': {
```

Comment:
For context: we've only recently started using `@emotion`—the sites management page is our first project using it. So we're still figuring out the patterns we like.

Fwiw we haven't been using nested class selectors like this in Emotion. It's perfectly valid CSS but I guess it works around the purpose of Emotion: defining your own class names and breaking that strong component-level style encapsulation. Given that dismiss button styles for both the single and double tile banner are identical it probably makes sense to define those styles as part of the `DismissButton` component.

Reply:
>Fwiw we haven't been using nested class selectors like this in Emotion. It's perfectly valid CSS but I guess it works around the purpose of Emotion: defining your own class names and breaking that strong component-level style encapsulation

I totally agree! I haven't used `@emotion` much and was on the `styled-components` side instead. Here's what you can do in `styled-components` to reference children:

```js
[ `${ Banner.DismissButton }` ]: { // dynamic class name gets inserted here
    position: 'absolute',
    top: 16,
    insetInlineEnd: 16,
},
```
This doesn't work with `@emotion`, and the best I found was [here](https://www.matthewsessions.com/blog/emotion-nested-selectors/), which is a bit more cumbersome. Maybe there is a better way?

>Given that dismiss button styles for both the single and double tile banner are identical it probably makes sense to define those styles as part of the DismissButton component.

I tend to prefer having everything in one place rather than overriding stuff. Saving a few lines of code seems good but makes it harder for the next person. Now I only use base styles if they are common to all instances, but the `row` banner uses different styles and would need to override the base styles, which I think makes the code harder to read. 

Dan Abramov wrote an article about "clean code" [here](https://overreacted.io/goodbye-clean-code/). It's a good read. 

On the subject of `styled` libraries. It's difficult to overlook stitches in 2022. It has very little runtime overhead and great theming and variant support. 

https://stitches.dev/

Reply:
Cool! I hadn't heard of stitches. The marketing claims at least sound great 😄 
We went with Emotion because it's what's being used by core WordPress, so probably unlikely we'd go in a different direction from them.

Personally I haven't been fully convinced by css-in-js. My impression so far is that it's probably great for theming, and there are probably cool SSR things you can do with it e.g. only send the CSS rules you need, but it doesn't add enough over and above plain CSS in the contexts that I've used it.